sdk/samples/iot/provisioning/src/paho_iot_provisioning_example.c

```diff
@@ -381,24 +381,24 @@ static int get_operation_status()
   // Successful assignment - print out the assigned hostname and device id
   if (operation_status == AZ_IOT_PROVISIONING_STATUS_ASSIGNED)
   {
-    printf("SUCCESS - Device provisioned:\n");
+    (void)printf("SUCCESS - Device provisioned:\n");
     print_az_span("\tHub Hostname: ", response.registration_result.assigned_hub_hostname);
     print_az_span("\tDevice Id: ", response.registration_result.device_id);
   }
   else // Unsuccesful assignment (unassigned, failed or disabled states)
   {
-    printf("ERROR - Device Provisioning failed:\n");
```

Comment:
Nit: We could extract methods for the success vs failure cases (maybe in the future?).